webplugin/template/mck-sidebox.html

```diff
@@ -281,15 +281,9 @@ <h3>No more messages!</h3>
 				<div class="mck-csat-wrapper n-vis" id="csat-2">
 				
 					<div class="mck-feedback-text-wrapper ">
-						<textarea name="feedback" cols="34" maxlength="90" id="mck-feedback-comment" placeholder="Add a comment…" aria-label="Add a comment..." tabindex="0" rows="4"></textarea>
-						<div class="mck-send-feedback" id="mck-submit-comment" aria-disabled="false" aria-label="Send feedback" tabindex="0">
-							<span class="mck-icon-send">
-								<svg focusable="false" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 15 15">
-									<path class="km-custom-widget-fill" fill="#5553B7" fill-rule="evenodd"
-										d="M7.878 7.858l-6.032.584-1.588 5.076a.787.787 0 0 0 1.097.944l13.22-6.401a.752.752 0 0 0 0-1.353L1.364.302a.787.787 0 0 0-1.096.944l1.587 5.077 6.021.584a.475.475 0 0 1 0 .946l.002.005z">
-									</path>
-								</svg>
-							</span>
+						<textarea name="feedback" cols="34" maxlength="90" id="mck-feedback-comment" placeholder="Add a feedback (optional)" aria-label="Add a feedback" tabindex="0" rows="4"></textarea>
```

Comment:
what is the purpose of aria-label? this needs to be same as placeholder? if yes, need to add (optional)

Reply:
its used for screen readers added the (optional)